ui/src/components/ExportFab.js

```diff
@@ -5,16 +5,30 @@ import "./ExportFab.css";
 import CloudUpload from "@material-ui/icons/CloudUpload";
 import Button from "@material-ui/core/Button";
 import Tooltip from "@material-ui/core/Tooltip";
+import TextField from '@material-ui/core/TextField';
```

Comment:
I get an error and warning when I run locally with your branch, does it work for you?